CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+22.0.0 - 2022-11-03
+-------------------
+- Add support and examples for Google Ads API v12.0.
+- Upgrade protobuf-java-util dependency to 3.19.6.
```

Comment:
Could you mention both protobuf-java and protobuf-java-util? They both went to 3.19.6.